### PR TITLE
docs(LoadBalancer): LoadBalancer and Kubernetes https error troubleshooting

### PR DESCRIPTION
**Pull request by thomasbeaujean**
# Description

When you have:
- In kubernetes, a pod wants to access a service within the same kube using https
- the https in handled by a Load Balancer configured for SSL Offload

Then a curl will not work, there will be an SSL error because the LB reroute the request before using the SSL.

### CODE REVIEW

File: network/load-balancer/troubleshooting/load-balancer-kubernetes-https-error.mdx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  paragraph: Fix the SSL Certificate when using https for services on the same kubernetes
+tags: load-balancer
+dates:
+  validation: 2024-06-23
+  posted: 2024-06-23
+categories:
+  - network
+---
+
+If your Load Balancer is created using kubernetes services, is configured for [SSL offload](/network/load-balancer/reference-content/ssl-bridging-offloading-passthrough/#configuring-a-load-balancer-for-ssl-offloading),
```
Comment (bene2k1) [inline network/load-balancer/troubleshooting/load-balancer-kubernetes-https-error.mdx:16]:
```suggestion
If your Load Balancer is created using Kubernetes services, is configured for [SSL offload](/network/load-balancer/reference-content/ssl-bridging-offloading-passthrough/#configuring-a-load-balancer-for-ssl-offloading),
```

File: network/load-balancer/troubleshooting/load-balancer-kubernetes-https-error.mdx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+and you have several services behind the same Load Balancer, you will have a SSL error when trying to reach the others services using https.
+
+Example:
+- podA
+- podB (https://serviceB.example.com)
```
Comment (bene2k1) [inline network/load-balancer/troubleshooting/load-balancer-kubernetes-https-error.mdx:21]:
```suggestion
- podB (`https://serviceB.example.com`)
```

File: network/load-balancer/troubleshooting/load-balancer-kubernetes-https-error.mdx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+From the podA, you want to curl https://serviceB.example.com, you will have a SSL Error.
```
Comment (bene2k1) [inline network/load-balancer/troubleshooting/load-balancer-kubernetes-https-error.mdx:23]:
```suggestion
From the podA, you want to curl `https://serviceB.example.com`, you will have an SSL Error.
```

**Comment by RoRoJ** [comment]
Thank you for this content. I think it would be more appropriate within the existing doc about Kubernetes Load Balancers, so I am closing this PR but will add your content to a new section on this page: https://www.scaleway.com/en/docs/containers/kubernetes/reference-content/kubernetes-load-balancer/

In the future we will review where the best place for Kuberentes x Load Balancer documentation is.

**Pull request closed without merging**